web-client/integration-tests/docketClerkVerifiesConsolidatedCases.test.js

```diff
@@ -0,0 +1,108 @@
+import { docketClerkAddsCaseToHearing } from './journey/docketClerkAddsCaseToHearing';
+import { docketClerkConsolidatesCases } from './journey/docketClerkConsolidatesCases';
+import { docketClerkCreatesATrialSession } from './journey/docketClerkCreatesATrialSession';
+import { docketClerkOpensCaseConsolidateModal } from './journey/docketClerkOpensCaseConsolidateModal';
+import { docketClerkRemovesCaseFromHearing } from './journey/docketClerkRemovesCaseFromHearing';
+import { docketClerkSealsCase } from './journey/docketClerkSealsCase';
+import { docketClerkSearchesForCaseToConsolidateWith } from './journey/docketClerkSearchesForCaseToConsolidateWith';
+import { docketClerkUnsealsCase } from './journey/docketClerkUnsealsCase';
+import { docketClerkUpdatesCaseStatusToReadyForTrial } from './journey/docketClerkUpdatesCaseStatusToReadyForTrial';
+import { docketClerkVerifiesConsolidatesCases } from './journey/docketClerkVerifiesConsolidatesCases';
+import { docketClerkViewsTrialSessionList } from './journey/docketClerkViewsTrialSessionList';
+import { loginAs, setupTest, uploadPetition } from './helpers';
+import { manuallyAddCaseToTrial } from './utils/manuallyAddCaseToTrial';
+import { petitionsClerkBlocksCase } from './journey/petitionsClerkBlocksCase';
+import { petitionsClerkPrioritizesCase } from './journey/petitionsClerkPrioritizesCase';
+import { petitionsClerkUnblocksCase } from './journey/petitionsClerkUnblocksCase';
+import { petitionsClerkUnprioritizesCase } from './journey/petitionsClerkUnprioritizesCase';
+
+const cerebralTest = setupTest();
+const trialLocation = `Boise, Idaho, ${Date.now()}`;
+
+const overrides = {
+  caseCaption: 'Mona Schultz, Petitioner',
+  docketNumberSuffix: 'L',
+  preferredTrialCity: trialLocation,
+  trialLocation,
+};
+
+describe('Case Consolidation Journey', () => {
+  beforeAll(() => {
+    jest.setTimeout(30000);
+  });
+
+  afterAll(() => {
+    cerebralTest.closeSocket();
+  });
+
+  cerebralTest.createdTrialSessions = [];
+
+  loginAs(cerebralTest, 'petitioner@example.com');
+
+  it('login as a petitioner and create the lead case', async () => {
+    const caseDetail = await uploadPetition(cerebralTest, overrides);
+    expect(caseDetail.docketNumber).toBeDefined();
+    cerebralTest.docketNumber = cerebralTest.leadDocketNumber =
+      caseDetail.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkUpdatesCaseStatusToReadyForTrial(cerebralTest);
+  docketClerkOpensCaseConsolidateModal(cerebralTest);
+  docketClerkSearchesForCaseToConsolidateWith(cerebralTest);
+
+  it('login as a petitioner and create the case to consolidate with', async () => {
```

Comment:
Are we still logged in as the docket clerk here?

Reply:
@mmarcotte resolved!

Reply:
Fixed to have the correct actor explicitly logging in at the appropriate steps!